etl/steps/data/garden/AI/2023-06-08/yougov_job_automation.meta.yml

```diff
@@ -0,0 +1,209 @@
+dataset:
+  title: How worried are Americans about being automated out of a job? (YouGov, 2023)
+  description: Biannual tracker on how worried are Americans about being automated out of a job? The provided results are weighted to ensure they are representative of the United States population across various demographic factors. The weighting is based on factors such as age, gender, race, education, region, political party affiliation, and income level. This weighting process helps to account for the diversity within the population and ensures that the results reflect a broader representation of the United States as a whole. Next update in July. Each wave comprises responses from 460 to 495 working US adults.
+  licenses:
+  - name: Public Data
+  sources:
+  - name: YouGov
+    description: Biannual tracker on how worried are Americans about being automated out of a job? The provided results are weighted to ensure they are representative of the United States population across various demographic factors. The weighting is based on factors such as age, gender, race, education, region, political party affiliation, and income level. This weighting process helps to account for the diversity within the population and ensures that the results reflect a broader representation of the United States as a whole. Next update in July. Each wave comprises responses from 460 to 495 working US adults.
+    url: https://today.yougov.com/topics/politics/trackers/worry-about-automation
+    source_data_url: https://today.yougov.com/_pubapis/v5/us/trackers/worry-about-automation/download/
+    date_accessed: '2023-06-08'
+    publication_date: '2023-01-12'
+    publication_year: 2023
+    published_by: YouGov (2023)
+tables:
+  yougov_job_automation:
+    variables:
+      _18_29:
+        title: 18-29
```

Comment:
The titles should be a one-line short and clear definition of the variable. "18-29" is definitely not clear (and the same applies to all other variables below).